### DISCUSSION TITLE
German translation

### DISCUSSION THREAD

**Discussion by Feldi77** [lllyasviel/Fooocus#2321] [Ideas]
Hello, I don't know if this is the correct procedure, but here is the German translation.
[de.json](https://github.com/lllyasviel/Fooocus/files/14363931/de.json)

**Comment by mashb1t** [comment]
@Feldi77 appreciate the effort! Please check for differences with https://github.com/lllyasviel/Fooocus/pull/1444 and add your comments there.

**Reply by Feldi77** [reply]
Hello, German is my mother tongue and I have so many deviations or improvements that it would be too much work to add them here as changes (picture shows only one screen, left is mine). I've been using mine for a while now and I haven't noticed any errors. What you do with it is up to you.

Best regards

![image](https://github.com/lllyasviel/Fooocus/assets/23531655/be61cebc-2c49-41cb-b8a9-e768cac3f067)
